### DISCUSSION TITLE
create test env for min or dev versions of dependencies

### DISCUSSION THREAD

**Discussion by davidism** [tox-dev/tox-uv#133] [Q&A] [answered]
I'm trying to convert Flask to use uv and tox-uv. Currently, we use pip-compile to create requirements files for different test environments. Our basic `testenv` runs `pytest` for each supported version of Python. But we also want to run our tests against the lowest supported versions and current dev versions of our dependencies (werkzeug, jinja, etc). So we have `py13-min` and `py39-dev` envs as well. https://github.com/pallets/flask/blob/6b054f8f3876ff4c31580b014d344c4cf491059d/tox.ini#L19-L21

I'm not sure how to accomplish this with uv and tox-uv. I still want to use uv to install the locked `tests` dependency group, but then I also want to install Flask's dependencies with `resolution = lowest` or with `deps = https://github.com...` entries. 

Do you use any environments like this? I figure this probably needs to get into uv itself, but maybe you have an idea of how to accomplish it with tox-uv now.

**Comment by gaborbernat** [comment, marked as answer]
Yeah, never tried doing something like that. I don't think you can do something like different resolutions logics per groups 🤔
